src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -491,19 +491,38 @@ async function prepareOCICache(dstFolder: string) {
 	return ociCacheDir;
 }
 
-function featuresToArray(config: DevContainerConfig): DevContainerFeature[] | undefined {
-	if (!config.features) {
+function featuresToArray(config: DevContainerConfig, additionalFeatures?: Record<string, string | boolean | Record<string, string | boolean>>): DevContainerFeature[] | undefined {
```

Comment:
Please make `additionalFeatures` a mandatory argument here and everywhere else where we want to make sure we don't forget pass it on.

Reply:
Done 😄